evalperf.html

```diff
@@ -300,6 +352,184 @@ <h2 id="sponsor" class="text-nowrap mt-5">🤗 Acknowledgment</h2>
+
+    let selectedXModels = modelData.slice(0, defaultDisplayNum).map(m => m.id);
+    let selectedYModels = modelData.slice(0, defaultDisplayNum).map(m => m.id);
+
+    function createCheckboxes() {
+      xAxisSelectors.innerHTML = '';
+      yAxisSelectors.innerHTML = '';
+
+      modelData.forEach(model => {
+        const xDiv = document.createElement('div');
+        xDiv.className = 'form-check';
+        xDiv.innerHTML = `
+          <div class="d-flex align-items-center" style="min-width: 0;">
+            <input class="form-check-input flex-shrink-0" type="checkbox" id="x-${model.id}"
+                  ${selectedXModels.includes(model.id) ? 'checked' : ''} >
+            <label class="form-check-label flex-grow-1 fs-6" for="x-${model.id}" title="${model.name}">
+              &nbsp${model.name}
+            </label>
+          </div>
+        `;
+        xAxisSelectors.appendChild(xDiv);
+
+        const yDiv = document.createElement('div');
+        yDiv.className = 'form-check';
+        yDiv.innerHTML = `
+          <div class="d-flex align-items-center" style="min-width: 0;">
+            <input class="form-check-input flex-shrink-0" type="checkbox" id="y-${model.id}"
+                  ${selectedYModels.includes(model.id) ? 'checked' : ''} >
+            <label class="form-check-label flex-grow-1 fs-6" for="y-${model.id}" title="${model.name}">
+              &nbsp${model.name}
+            </label>
+          </div>
+        `;
+        yAxisSelectors.appendChild(yDiv);
+      });
+
+      document.querySelectorAll('#xAxisSelectors input[type="checkbox"]').forEach(checkbox => {
+        checkbox.addEventListener('change', function() {
+          const modelId = this.id.slice(2);
+          if (this.checked) {
+            selectedXModels.push(modelId);
+          } else {
+            selectedXModels = selectedXModels.filter(id => id !== modelId);
+          }
+          updateCheckboxStates();
+          updateHeatmap();
+        });
+      });
+
+      document.querySelectorAll('#yAxisSelectors input[type="checkbox"]').forEach(checkbox => {
+        checkbox.addEventListener('change', function() {
+          const modelId = this.id.slice(2);
+          if (this.checked) {
+            selectedYModels.push(modelId);
+          } else {
+            selectedYModels = selectedYModels.filter(id => id !== modelId);
+          }
+          updateCheckboxStates();
+          updateHeatmap();
+        });
+      });
+    }
+
+    function updateCheckboxStates() {
+    }
+
+    function updateHeatmap() {
+      const xModels = modelData.filter(m => selectedXModels.includes(m.id));
+      const yModels = modelData.filter(m => selectedYModels.includes(m.id)).reverse();
+
+      const heatmapData = [];
+      yModels.forEach((model1, i) => {
+        xModels.forEach((model2, j) => {
+          const diff = model1.dps - model2.dps;
+          heatmapData.push([j, i, model1.id !== model2.id ? parseFloat(diff.toFixed(1)) : '']);
```

Comment:
I think the number we want to visualize is like `${Examinee's DPS} (${Diff towards the  reference})`